src/components/table/makeSelectable.js

```diff
@@ -98,118 +102,259 @@ function makeSelectable(BaseTable) {
             }
         };
 
+        getSharedHotkeyConfigs = () => {
+            const { hotkeyType } = this.props;
+            return [
+                new HotkeyRecord({
+                    key: 'shift+x',
+                    description: <FormattedMessage {...messages.shiftXDescription} />,
+                    handler: () => {
+                        const { focusedIndex } = this.state;
+
+                        if (focusedIndex === undefined) {
```

Comment:
It's interesting that `clearFocus` sets `focusedIndex` to `undefined` instead of `null`. Some people think that it was a mistake to include both in JS.

Reply:
Hmm, I would have expected clearFocus to assign `null`, but maybe I don't understand how `null` is intended to be used.